src/pytorch_lightning/loggers/mlflow.py

```diff
@@ -240,20 +241,26 @@ def experiment_id(self) -> Optional[str]:
     def log_hyperparams(self, params: Union[Dict[str, Any], Namespace]) -> None:
         params = _convert_params(params)
         params = _flatten_dict(params)
+        # params = [Param(k, v) for k, v in params.items()]
+        params_list: List[Param] = []
+
         for k, v in params.items():
+            # FIXME: mlflow 1.28 allows up to 500 characters: https://github.com/mlflow/mlflow/releases/tag/v1.28.0
```

Comment:
Are you planning to address this in this PR? I suggest to do it separately.

Reply:
I wasn't planning to address this in this PR, but figured adding a FIXME comment would still be appreciated for future devs — or should I remove it?

I'd be happy to address this in its own PR, though I'd have to study the patterns in place for managing logic conditional on version introspection.

Reply:
```suggestion
            # TODO: mlflow 1.28 allows up to 500 characters: https://github.com/mlflow/mlflow/releases/tag/v1.28.0
```